src/index.html

```diff
@@ -1,13 +1,67 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image image-iMac"
+        src="images/imac.jpeg"
+        alt="APPLE iMac"
+      >
+
+      <div class="card__text">
+        APPLE A1419 iMac 27&#34; Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__code">Product code: 195434</div>
+
+      <div class="card__stars">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="card__stars-reviews">
+          Reviews: 5
+        </div>
+      </div>
+
+      <div class="card__price prise-inside">
+        <div class="prise-inside__text">
+          Price:
+        </div>
+
+        <div class="prise-inside__value">
+          &#36;2,199
+        </div>
+      </div>
+
+      <a
+         href="/"
+         data-qa="hover"
+        class="card__link-buy link-buy"
+      >
```

Comment:
```suggestion
      <a
       href="/"
       data-qa="hover"
       class="card__link-buy link-buy"
      >
```
Check the indentations, please.